.eslintrc.js

```diff
@@ -43,6 +43,10 @@ module.exports = {
       }
     }],
     'one-var': ['error', 'never'],
+    'import/no-extraneous-dependencies': ['error', {
+      /* the following folders can import devDependencies */
+      devDependencies: ['test/**', 'tests/**', 'src/debug/**', 'examples/**'],
```

Comment:
Wouldn't one goal be to make the sources usable "as-is", without webpack preprocessing? (or even simpler, and could –almost– be done today already: after conversion to ES5 through Babel)

Then developers using iTowns2 through NPM could choose to enable debug or not **in their projects**, and would have issues with the code not finding some dependencies. Now move those to `optionalDependencies` and everything's OK: want to enable debug? then add this dependency from our set of optional dependencies.

Now maybe I'm mistaken and that doesn't make sense; still it "smells" IMO that something in `src/` could depend on dev dependencies.